### ISSUE TITLE
Change "just now" label to "now"

### ISSUE THREAD

**Issue by silkfire** [catamphetamine/javascript-time-ago#38]
In a recent version, the `twitter` style label "now" was changed to "just now". Is there any way I could customize my style to use the previous wording?

**Comment by catamphetamine** [comment]
As a workaround, en[‘mini-time’].now = “now” wold work.
I’ll address the issue tomorrow.

**Comment by catamphetamine** [comment]
Fixed in `javascript-time-ago@2.2.5`: now it outputs "now" for 0 seconds and "1s" for 1 second.

**Comment by catamphetamine** [comment]
@silkfire Update: Added `"twitter-now"` style that outputs `"now"` for `0` seconds. `"twitter"` style doesn't output `"now"` now.

**Comment by silkfire** [comment]
I think I probably was confused myself about what I was looking for when I opened this issue in the first place.
What I actually wanted was to get the old behavior that I had with my setup.

I want the steps (gradatation?) of the old `twitter` style, so basically everything <1 min shows as 'now' (no 1-59s), after that it should show as 1m, 2h, 3d, 4m, 5y, etc. But I don't want the rounding to happen. 8 min should be 8 min and not 10. Is there any way to achieve this behavior?

**Comment by catamphetamine** [comment]
How about `0-40s` — "now", then "1m", "2m", etc.

**Comment by silkfire** [comment]
I'd be okay with that. Is this achievable in the current version?

**Comment by catamphetamine** [comment]
> I'd be okay with that. 

Not ideal or good?

**Comment by silkfire** [comment]
> > I'd be okay with that.
> 
> Not ideal or good?

Well I wasn't expecting 40-59s to be rounded up to 1m but I could be okay with that if the rest works as it did before.

**Comment by catamphetamine** [comment]
> Well I wasn't expecting 40-59s to be rounded up to 1m but I could be okay with that if the rest works as it did before.

That's the question: what's better.

```
0-30 — now
30-90 — 1m
90-150 — 2m
etc
```

or

```
0-60 — now
60-90 — 1m // <--- disproportionately small
90-150 — 2m
etc
```

I'd prefer the first one.
You?

**Comment by catamphetamine** [comment]
See `"twitter-minute-now"` style in `javascript-time-ago@2.2.8`.

**Comment by silkfire** [comment]
> > Well I wasn't expecting 40-59s to be rounded up to 1m but I could be okay with that if the rest works as it did before.
> 
> That's the question: what's better.
> 
> ```
> 0-30 — now
> 30-90 — 1m
> 90-150 — 2m
> etc
> ```
> 
> or
> 
> ```
> 0-60 — now
> 60-90 — 1m // <--- disproportionately small
> 90-150 — 2m
> etc
> ```
> 
> I'd prefer the first one.
> You?

What's wrong with 0-59 — now, 60-119 — 1m, etc?

**Comment by catamphetamine** [comment]
> What's wrong with 0-59 — now, 60-119 — 1m, etc?

Ah, I see.
The currently implemented rounding mechanism performs a `Math.round()` when calculating the amount of time units, so 90 seconds is rounded to 2 minutes.
Perhaps there could be a place for "custom rounding", like `Math.floor()` instead of `Math.round()`.
We'll see if people complain. If they do, then maybe I could implement a `round: 'round'/'floor'` option

**Comment by silkfire** [comment]
That'd be nice to give the developer more control over the rounding mechanics, as of right now it seems very arbitrary to me.

**Comment by catamphetamine** [comment]
Published `javascript-time-ago@2.3.0`.
It has `round` parameter on a style or on `.format()`.
Can be: "round", "floor".
